Index.js

```diff
@@ -1,13 +1,17 @@
 <!DOCTYPE html>
 <html>
 <body>
-<p id="demo">he value of z is.</p>
+<p id="demox">he value of x is.</p>
```

Comment:
Can Use better Variables.
demoxx
demoyy
demozz

Reply:
I am going to do that.